rundeckapp/grails-spa/packages/ui-trellis/src/components/execution-log/logViewer.vue

```diff
@@ -478,9 +478,14 @@ export default class LogViewer extends Vue {
     }
 
     private handleNewLine(entries: Array<any>) {
+       const updatableProps: Array<keyof IEventViewerSettings> = ['nodeBadge', 'gutter', 'timestamps', 'lineWrap', 'command']
```

Comment:
Is this a dupe of the set in `@Watch('settingsChange'`?

Reply:
Removed